### ISSUE TITLE
Python naming conventions

### ISSUE THREAD

**Issue by jayvdb** [storyscript/community#17]
tl:dr; PEP 8 module naming was suggested at https://github.com/storyscript/hub-sdk-python/issues/9 , and a counter argument was provided.  Decision needs to be made soon, globally, and then enforced and not changed for "a long time".

In short, PEP 8 naming is snake_case filenames, which are module names.  Storyscript code bases do not use this convention.  Good tools exist in Python world for snake_case.  Tools for our current naming convention are not so good.

There is an unknown in play, which is that `black` (https://github.com/storyscript/community/issues/16) might start making opinionated decisions which conflict with any choice here.  They are known to cut against the PEP8 grain with well justified decisions, so I wouldnt want to bet that they would enforce PEP8 naming.  However they are likely to be sane enough to allow the module naming conventions to be disabled.

**Comment by williammartin** [comment]
I'm in agreement that we should make the decision asap. I don't feel strongly about any of the proposed solutions as I've not been in the python ecosystem long enough to form them.

If you pushed me for an opinion I'd suggest using PEP8 everywhere because it's well supported.

Someone come tell me why I'm wrong.
